react/components/SKUSelector/components/SKUSelector.tsx

```diff
@@ -366,4 +366,4 @@ const SKUSelector: FC<Props> = ({
   )
 }
 
-export default memo(SKUSelector)
+export default Object.assign(memo(SKUSelector), { cssHandles: CSS_HANDLES })
```

Comment:
I think that is more easy to understand, what do you think?

```suggestion
const MemoizedSKUSelector = memo(SKUSelector)

MemoizedSKUSelector.cssHandles = CSS_HANDLES

export default MemoizedSKUSelector
```

Reply:
I agree, but the types of React.memo didn't allow me to do this.